test/test_hip_rdna3.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+import unittest
+from tinygrad import Tensor
+from tinygrad.helpers import getenv
+from tinygrad.ops import LoadOps
+from tinygrad.codegen.kernel import LinearizerOptions
+from tinygrad.codegen.linearizer import Linearizer
+from tinygrad.renderer.hip import HIPRenderer
+from examples.beautiful_mnist import Model
+import extra.hip_wrapper as hip
+
+
+class TestHIPCompilationRDNA(unittest.TestCase):
+  def test_compile_hip(self):
+    if not getenv("HIP"): raise unittest.SkipTest("testing HIP->rdna3 compilation needs HIP=1")
+
+    model = Model()
+
+    input = Tensor.rand(512,1,28,28, device="CPU").numpy()
+    result = model(Tensor(input))
```

Comment:
can't you just use a tinygrad tensor instead of converting it numpy and then convert back?

Reply:
oops. forgot it there from a poor attempt at fixing it failing on the rand op (the right was to filter off LoadOps above)